pom.xml

```diff
@@ -45,7 +45,18 @@
           <configuration>
             <excludeFilterFile>src/findbugs/excludesFilter.xml</excludeFilterFile>
             <failOnError>${maven.findbugs.failure.strict}</failOnError>
+            <xmlOutput>true</xmlOutput>
+            <findbugsXmlWithMessages>true</findbugsXmlWithMessages>
```

Comment:
These messages provide context details. E.g.: "Access to field foo is not synchronized. Synchronized 66% times"